### ISSUE TITLE
[SF_ACC] [TabContainer] Sub-tabs Down-Arrow Missing Alt Text

### ISSUE THREAD

**Issue by IrvinDelgado** [SAP/ui5-webcomponents#6001]
### **Bug Description**
In TabContainer when the component contains nested tabs a Down-Arrow Icon is shown however, the button does not contain alternate-text

### **Expected Behavior**
To expect all icons to have alternate-text

### **Steps to Reproduce**
1. Go to [TabContainer](https://sap.github.io/ui5-webcomponents/playground/components/TabContainer/)
2. Inspect the TabContainer with nested tabs and observe that there is no alternate text

### **Context**
- UI5 Web Components version: v1.9.0
- OS/Platform: MacOs Monterey
- Browser: Chrome
- Affected component: TabContainer

### **Log Output / Stack Trace / Screenshots**

<img width="392" alt="TabConainer_Missing_AltText" src="https://user-images.githubusercontent.com/62973847/199609823-ba8bc8e4-f336-4c61-8e7f-873e8db0544c.png">

### **Priority**
- [ ] Low
- [x] Medium
- [ ] High
- [ ] Very High

The priority indicates the severity of the issue. To set the appropriate priority consider the following criteria:
- Breaks entire application or system - High or Very High
- Accessibility issue - Medium or High
- Functional issue - Medium or High
- Visual issue - Low or Medium

**Note:** The priority might be re-evaluated by the issue processor.

### **Stakeholder Info** (if applicable)

- Organization: WEF

**Comment by TeodorTaushanov** [comment]
Hi,

The issue is fixed in the main branch.

Best,
Teo
